app/src/main/java/com/x8bit/bitwarden/data/auth/repository/model/PolicyInformation.kt

```diff
@@ -61,8 +61,8 @@ sealed class PolicyInformation {
      */
     @Serializable
     data class PasswordGenerator(
-        @SerialName("defaultType")
-        val defaultType: String?,
+        @SerialName("overridePasswordType")
```

Comment:
Did this change since it's initial implementation?

Reply:
yes, both name and functionality have been updated recently

https://github.com/bitwarden/android/pull/3708#discussion_r1713891587